src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -247,36 +287,42 @@ def _update_onpen_bar(self):
         self._update_progress_event(ev_id=self.onpen_pev_id, ev_msg=msg,
                                     ev_progress_fraction=avg_progress_fraction)
 
-    def _update_ongoing_bar(self):
+    def _update_ongoing_bar(self, clones):
```

Comment:
Right. But we have a problem. When I launch 4 clones in one go using automation, ctime for many of those clones are exactly same, which doesn't allow sorting based on ctime. Is it possible to get microseconds/milliseconds in ctime? 

I've copied an example here in comments - https://github.com/ceph/ceph/pull/54620/files#diff-32b8a30d8ea6ea9fd5b4753c03bfa92c14dd72aab44cdb973856448514d0b320R76

Reply:
Maybe python stat does not provide that granularity that you are looking for. Please check.

OTOH, is that really required from a functionality POV? For testing, you could add a small delay before scheduling clones. Would that work?

Reply:
Files -
```
$ touch d; touch e; touch f
```

Stating file above using GNU coreutil's `stat` -
```
Birth: 2024-03-05 16:53:33.685796732 +0530
Birth: 2024-03-05 16:53:33.686796734 +0530
Birth: 2024-03-05 16:53:33.687796736 +0530
```

Stating using Python's `os.stat` -
```
>>> st_b_d.st_ctime
1709637813.6857967
>>> st_b_e.st_ctime
1709637813.6867967
>>> st_b_f.st_ctime
1709637813.6877968

>>> datetime.datetime.fromtimestamp(st_b_d.st_ctime)
datetime.datetime(2024, 3, 5, 16, 53, 33, 685797)
>>> datetime.datetime.fromtimestamp(st_b_e.st_ctime)
datetime.datetime(2024, 3, 5, 16, 53, 33, 686797)
>>> datetime.datetime.fromtimestamp(st_b_f.st_ctime)
datetime.datetime(2024, 3, 5, 16, 53, 33, 687797)
```

Perhaps there is a way to get those last 2 digits as well.

Reply:
> OTOH, is that really required from a functionality POV? For testing, you could add a small delay before scheduling clones. Would that work?

We can do so in testing but in real scenario if multiple clones are launched using script it might lead to a bug. So, I am trying to stick to it.